Sources/VLCSettingsTableHeaderView.swift

```diff
@@ -0,0 +1,65 @@
+/*****************************************************************************
+ * VLCSettingsTableHeaderView.swift
+ * VLC for iOS
+ *****************************************************************************
+ * Copyright (c) 2018 VideoLAN. All rights reserved.
+ * $Id$
+ *
+ * Authors: Mike JS. Choi <mkchoi212 # icloud.com>
+ *
+ * Refer to the COPYING file of the official project for license.
+ *****************************************************************************/
+
+import UIKit
+
+class VLCSettingsTableHeaderView: UITableViewHeaderFooterView {
+    @objc static let identifier = String(describing: VLCSettingsTableHeaderView.self)
+    @objc var headerLabel: UILabel!
+    
+    override var frame: CGRect {
```

Comment:
This was required to silence auto layout warnings. See https://stackoverflow.com/a/35053234/4064189

Reply:
could you try giving the constraints a priority of 999. This allows them to break temporarily while the height or width is 0